### PR TITLE
amend some small mistakes on annotation

### PR DESCRIPTION
**Pull request by WanLinghao**
modified:   pkg/kubelet/dockershim/libdocker/client.go
modified:   staging/src/k8s.io/apiserver/pkg/storage/etcd/util/doc.go

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by resouer** [comment]
I believe this is due to test flake. But the fix make sense.

/lgtm

**Review by resouer — APPROVED** [review]

**Comment by cblecker** [comment]
@WanLinghao `pull-kubernetes-e2e-kops-aws` has a known failure at the moment. please disregard.

**Comment by WanLinghao** [comment]
@yujuhong @xiang90  PTAL

File: pkg/kubelet/dockershim/libdocker/client.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// https://docs.docker.com/engine/reference/api/docker_remote_api/
-	// docker version should be at least 1.10.x
+	// docker version should be at least 1.22.x
```
Comment (yujuhong) [inline pkg/kubelet/dockershim/libdocker/client.go]: This is not a mistake. Docker 1.10.x uses the 1.22 API.

**Review by resouer — CHANGES_REQUESTED** [review]
Close this due to misunderstanding.

**Pull request closed without merging**